haiku-apps/projectconceptor/projectconceptor-0.1.1.recipe

```diff
@@ -55,7 +61,9 @@ defineDebugInfoPackage projectconceptor \
 	"$appsDir"/$APP/Plugins/RemoveAttribute \
 	"$appsDir"/$APP/Plugins/Resize \
 	"$appsDir"/$APP/Plugins/Select \
-	"$appsDir"/$APP/ProjectConceptor
+	"$appsDir"/$APP/ProjectConceptor \
+	"$addOnsDir"/Translators/FreeMindTranslator \
+	"$addOnsDir"/Translators/ProjectConceptorTranslator
```

Comment:
I think the quotes areound $appsDir and $addOnsDir aren't needed. At least they aren't used elsewhere in this or other recipes.